apps/docusaurus/docs/move/move-on-aptos/move-security-guidelines.md

```diff
@@ -12,7 +12,65 @@ This guide addresses this gap by detailing common anti-patterns and their secure
 
 ---
 
-### Object Access Control
+### Object Ownership Check
+
+Every `Object<T>` can be accessed by anyone, which means any `Object<T>` can be passed to any function, even if the caller doesn't own it.
+It's important to verify that the `signer` is the rightful owner of the object.
```

Comment:
This isn't really any different than a resource as well?  Maybe we should mention that